bank-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -27,3 +27,29 @@ describes what has changed.
 Please be aware that for the Gini Bank SDK you need to use the ``CaptureConfiguration`` instead of the Gini Capture SDK's
 ``GiniCapture.Builder``. The configuration names are the same so you can easily map them to the
 ``CaptureConfiguration``. This applies also to the new configuration options added in 3.0.0.
+
+
+Migrate the Return Assistant
+-----------------------------------
+
+Onboarding
+~~~~~~~~~~
+
+The new onboarding screen uses the global UI customization options. You can discard the old screen specific
+customizations.
+
+`Here <customization-guide.html#onboarding-screen>`_ you can find the detailed description on how to customize this screen.
+
+Breaking Changes
+++++++++++++++++
+
+Old UI is replaced with new UI.
+
+New Features
```

Comment:
[Comment] Let's wait until the image container injection PR (#201) is approved and then add a short description about it here.